### ISSUE TITLE
Impove keepass2android keybard

### ISSUE THREAD

**Issue by zveber** [PhilippC/keepass2android#1187]
For some reason I use the largest font on my phone. The problem is that captions 'username' and 'password' look quite bad.
@
![Screenshot_2020-03-10-20-04-55-454_joa zipper editor](https://user-images.githubusercontent.com/744888/76342522-cccf0500-630f-11ea-97a8-0b2d44b76a16.jpg)
So it would be nice if you use some icons instead. For example
* https://feathericons.com/?query=user
* https://feathericons.com/?query=Key

**Comment by NumbGnat** [comment]
I agree. Many times I have security questions & answers and I can't tell which item is which.  Perhaps an alternative selection method?

Instead of the [>] right arrow button to cycle through the settings, perhaps an [^] up arrow button that would pop up a listbox of possible fields (other than username & password).

I do think adopting some universal icons for the username and password buttons, like @zveber suggested, would help reduce the footprint of the keyboard.
